drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -189,7 +165,7 @@ def create_astrometric_catalog(inputs, **pars):
     for source in ref_dict:
         if 'GAIAsourceID' in source:
             g = source['GAIAsourceID']
-            if gaia_only and g.strip() is '':
+            if gaia_only and g.strip() == '':
                 continue
         else:
             g = -1  # indicator for no source ID extracted
```

Comment:
As noted by @mcara , the value of `g` should be set here to `"-1"`, since the table row has that entry defined with dtype of `U25`.  The code will work with or without this change, though, since astropy `Table()` objects have no problems with the values either way.